test/functional/p2p_segwit.py

```diff
@@ -488,8 +489,7 @@ def test_v0_outputs_arent_spendable(self):
 
         # Create two outputs, a p2wsh and p2sh-p2wsh
         witness_program = CScript([OP_TRUE])
```

Comment:
Unrelated comment in 905d672b743edf31531d095ffe800449eaffec69:

This is wrong. This is the *preimage* to the witness program. According to BIP 141:

> ... 1-byte push opcode (for 0 to 16) followed by a data push between 2 and 40 bytes gets a new special meaning. The value of the first push is called the "version byte". The following byte vector pushed is called the "witness program". 

The proper name is witnessScript:

> The last item in the witness (the "witnessScript") is popped off, hashed with SHA256 ...

Reply:
Being updated in #22429.